gdrivefs/gdfs/gdfuse.py

```diff
@@ -852,6 +852,10 @@ def mount(auth_storage_filepath, mountpoint, debug=None, nothreads=None,
             else:
                 v = True
 
+            # ignore the "user" option so we can put in fstab easily
+            if k == "user":
```

Comment:
I'm not clear on why we need this. If we receive an unsupported option it's just forwarded to FUSE and there is no such "user" option. So, it should work fine without this change.